docs/services/slam/cartographer/_index.md

```diff
@@ -13,93 +13,107 @@ aliases:
 
-The `cartographer` {{< glossary_tooltip term_id="module" text="module" >}} is available [from the Viam registry](https://app.viam.com/module/viam/cartographer).
-See [Modular resources](/modular-resources/#the-viam-registry) for instructions on using a module from the Viam registry on your robot.
+## Requirements
 
-The source code for this module is available on the [`viam-cartographer` GitHub repository](https://github.com/viamrobotics/viam-cartographer).
+- Your robot must have an RPlidar installed to be able to use the `cartographer` module, such as the [RPlidar A1](https://www.slamtec.com/en/Lidar/A1) or [RPlidar A3](https://www.slamtec.com/en/Lidar/A3).
 
-{{% alert title="Info" color="info" %}}
+- In addition, you must [add the `rplidar` module to your robot](/modular-resources/examples/rplidar/) to support the RPlidar hardware, if you have not done so already.
 
-Cartographer supports taking **2D LiDAR** or **3D LiDAR** data and optionally **inertial measurement unit (IMU)** and/or **odometry** data as input.
+  {{< alert title="SUPPORT" color="note" >}}
 
-However, currently, the `cartographer` modular resource only supports taking **2D LiDAR** and optionally **IMU** data as input. Support for taking **3D LiDAR** and **odometry** data as input may be added in the future.
+  Currently, the `rplidar` and `cartographer` modules support the Linux platform only.
 
-{{% /alert %}}
+  {{< /alert >}}
 
-## Online mode
+- You must physically connect your RPlidar to your robot before configuring the `rplidar` and `cartographer` modules.
 
-In this mode, you use **live** LiDAR and optional IMU data to **create a map**, **update an existing map**, or **do pure localization** on an existing map.
+  If you have a Viam Rover and are mounting an RPlidar to your rover, be sure to position the RPlidar so that it **faces forward in the direction of travel**, facing in the same direction as the included webcam.
+  For example, if you are using the [RPlidar A1](https://www.slamtec.com/en/Lidar/A1) model, mount it to the Rover so that the **pointed** end of the RPlidar mount housing points in the direction of the front of the Rover.
 
-### Requirements
-
-If you haven't already, [install `viam-server`](/installation/) on your robot.
+  If you need a **mount plate** for your RPlidar A1 or A3 model, you can 3D print an adapter plate using the following:
 
-Your robot must have an RPlidar installed to be able to use the `cartographer` module, such as the [RPlidar A1](https://www.slamtec.com/en/Lidar/A1) or [RPlidar A3](https://www.slamtec.com/en/Lidar/A3).
+  - [RPlidar A1 adapter STL](https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidarA1_adapter.STL)
+  - [RPlidar A3 adapter STL](https://github.com/viamrobotics/Rover-VR1/blob/master/CAD/RPIidarA3_adapter.STL)
 
-In addition, you must [add the `rplidar` module to your robot](/modular-resources/examples/rplidar/) to support the RPlidar hardware, if you have not done so already.
+## Mapping Modes
```

Comment:
I think one thing that's confusing is in this "Mapping Modes" section, we're describing the three modes and then showing you (and linking to) how to run them with a _live_ robot. But you can also create and update maps with previously captured data.

Reply:
Changed this to be different headers. Let me know if that makes sense (still work in progress)

Reply:
These make sense, but I still slightly feel like it'd be clearest to nest the "Create a new map", "Update existing map", and "Localize only" headers/sections under something that says it's for using a live robot, since the first two also work in "previously captured data" mode